pkg/helpers/helpers.go

```diff
@@ -189,10 +189,15 @@ func CreateSaveSSH(username, outputDirectory string, s *i18n.Translator) (privat
 
 // GetCloudTargetEnv determines and returns whether the region is a sovereign cloud which
 // have their own data compliance regulations (China/Germany/USGov) or standard
-//  Azure public cloud
-func GetCloudTargetEnv(location string) string {
+// Azure public cloud
+// CustomCloudName is name of environment if customCloudProfile is provided, it will be empty string if customCloudProfile is empty.
+// Because customCloudProfile is empty for deployment for AzurePublicCloud, AzureChinaCloud,AzureGermanCloud,AzureUSGovernmentCloud,
+// The customCloudName value will be empty string for those clouds
+func GetCloudTargetEnv(location, customCloudName string) string {
```

Comment:
I don't think we should change the method signature of a public method, given that we have other projects that vendor our codebase.

We should just refrain from calling this method  `GetCloudTargetEnv` altogether , if `customCloudName` is non-empty.